### ISSUE TITLE
Failed to load the project-runner package every time it start

### ISSUE THREAD

**Issue by lgrube** [hirohisa/atom-project-runner#8]
[Enter steps to reproduce below:]

1. ...
2. ...

**Atom Version**: 1.1.0
**System**: Mac OS X 10.11.1
**Thrown From**: [project-runner](https://github.com/hirohisa/atom-project-runner) package, v0.2.0

### Stack Trace

Failed to load the project-runner package

```
At Cannot read property 'prototype' of undefined

TypeError: Cannot read property 'prototype' of undefined
    at __extends (/Users/lukegrube/.atom/packages/project-runner/lib/project-runner-view.coffee:1:1)
    at /Users/lukegrube/.atom/packages/project-runner/lib/project-runner-view.coffee:7:3
    at Object.<anonymous> (/Users/lukegrube/.atom/packages/project-runner/lib/project-runner-view.coffee:7:3)
    at Object.<anonymous> (/Users/lukegrube/.atom/packages/project-runner/lib/project-runner-view.coffee:1:1)
    at Module._compile (module.js:428:26)
    at Object.defineProperty.value [as .coffee] (/Applications/Atom.app/Contents/Resources/app.asar/src/compile-cache.js:169:21)
    at Module.load (module.js:353:32)
    at Function.Module._load (module.js:308:12)
    at Module.require (module.js:363:17)
    at require (module.js:382:17)
```

### Commands

```
```

### Config

```json
{
  "core": {
    "audioBeep": false,
    "projectHome": "/Users/lukegrube/Documents/Atom/code",
    "themes": [
      "dracula-ui",
      "hydra-syntax-theme"
    ],
    "excludeVcsIgnoredPaths": false,
    "openEmptyEditorOnStart": false,
    "ignoredNames": [
      ".hg",
      ".svn",
      ".DS_Store",
      "._*",
      "Thumbs.db"
    ]
  }
}
```

### Installed Packages

```coffee
# User
atom-alignment, v0.12.1
css-color-underline, v1.0.1
css-comb, v2.4.0
css-snippets, v0.9.0
css-specificity, v0.2.2
dracula-ui, v0.3.0
file-icons, v1.6.11
git-log, v0.4.1
github-utils, v0.7.0
hydra-syntax-theme, v0.2.4
javascript-snippets, v1.0.0
jshint, v1.8.3
language-diff, v0.3.1
linter, v1.10.0
linter-gcc, v0.3.9
linter-jscs, v3.1.2
linter-jshint, v1.2.1
markdown-preview-plus, vundefined
pigments, v0.18.1
project-runner, v0.2.0
tree-view-git-status, v0.2.2

# Dev
No dev packages
```

**Comment by kolomiichenko** [comment]
:+1:

**Comment by hirohisa** [comment]
Sorry, I'm late. 🙇 

Fix by #9. Bump version to `0.3.0`, please update
